{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:26:58.899Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:27:01.796Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:27:03.500Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:27:05.826Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:28:04.811Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:28:06.020Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:30:09.310Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:30:12.337Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:30:21.009Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:30:21.562Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:30:23.387Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:35:11.961Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:35:13.193Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:38:41.156Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:38:43.044Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:38:43.748Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:39:58.793Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:43:50.677Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:225:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:45:39.180Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:225:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:48:15.434Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:225:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:48:39.303Z"}
{"code":"ETIMEOUT","connectionString":"mongodb+srv://\\1:****@webhlo.71edfez.mongodb.net/?retryWrites=true&w=majority&appName=webhlo","level":"error","message":"❌ MongoDB connection error: queryTxt ETIMEOUT webhlo.71edfez.mongodb.net","service":"redeem-api","timestamp":"2025-05-10T07:53:03.404Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:225:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T07:56:24.281Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:225:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T11:01:56.802Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:225:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T11:01:58.039Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:225:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T11:01:58.784Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:213:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T13:38:26.747Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:213:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T13:43:56.512Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:213:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T13:43:58.881Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:213:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T13:43:59.993Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:213:25\n    at Server.close (node:net:2315:9)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T13:44:48.019Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:216:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T13:47:42.910Z"}
{"level":"error","message":"❌ Unhandled promise rejection:","reason":{},"service":"redeem-api","stack":"MongooseError: Connection.prototype.close() no longer accepts a callback\n    at NativeConnection.close (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\node_modules\\mongoose\\lib\\connection.js:1205:11)\n    at Server.<anonymous> (C:\\Users\\stron\\OneDrive\\Documents\\WEBHLO\\redeem-backendwebhlo\\server.js:217:25)\n    at Object.onceWrapper (node:events:632:28)\n    at Server.emit (node:events:518:28)\n    at emitCloseNT (node:net:2375:8)\n    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-05-10T13:51:02.435Z"}
{"connectionString":"mongodb+srv://\\1:****@webhlo.71edfez.mongodb.net/?retryWrites=true&w=majority&appName=webhlo","level":"error","message":"Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://www.mongodb.com/docs/atlas/security-whitelist/","service":"redemption-api","timestamp":"2025-05-11 14:42:43"}
{"level":"error","message":"Kesalahan autentikasi: jwt expired","service":"redemption-api","timestamp":"2025-05-11 22:55:35"}
{"connectionString":"mongodb+srv://\\1:****@webhlo.71edfez.mongodb.net/?retryWrites=true&w=majority&appName=webhlo","level":"error","message":"Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://www.mongodb.com/docs/atlas/security-whitelist/","service":"redemption-api","timestamp":"2025-05-12 09:26:00"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:04:52"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:04:52"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:04:56"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:04:56"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:10:23"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:10:23"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:10:43"}
{"level":"error","message":"Kesalahan autentikasi: B45A0000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-12 22:10:43"}
{"level":"error","message":"Error pada login: Topology is closed","service":"redemption-api","timestamp":"2025-05-12 22:10:58"}
{"level":"error","message":"Error pada login: C4190000:error:0A000438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\rec_layer_s3.c:1605:SSL alert number 80\n","service":"redemption-api","timestamp":"2025-05-13 00:46:39"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:15"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:16"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:17"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:18"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:18"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:18"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:19"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:19"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:19"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:19"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:20"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:21"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:22"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:22"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:22"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:22"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:23"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:24"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:27"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:39:27"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:25"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:26"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:26"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:26"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:26"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:26"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:27"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:27"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:27"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:27"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:28"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:28"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:28"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:28"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:28"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:28"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:28"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:29"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:30"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:30"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:30"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:30"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:30"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:30"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:31"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:31"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:31"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:31"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:31"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:31"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:31"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:32"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:32"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:32"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:32"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:32"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:33"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:33"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:33"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:33"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:35"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:36"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:37"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
{"level":"error","message":"Kesalahan autentikasi: jwt malformed","service":"redemption-api","timestamp":"2025-05-13 11:40:38"}
